File: 1/1.go
package main

import (
	"os"
	"strconv"
)
func main(){
	s:=0
	const n = strconv.Atoi(os.Args[1])



	for i := range [n]int{} {
		s=s+(i+i*i^i)%(i+1)

}

	println(s)
}

File: 1/1.c
int main(int argc, char **argv)
{
    int s = 0;
    for (int i=0;i<=atoi(argv[1]);i++)
    {
        //printf("%d ->%d\n",i,s);
        s=s+(i+i*i^i)%(i+1); 
    }
    printf("%d\n",s);
}